core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -5350,6 +5396,8 @@ object ZIO extends ZIOCompanionPlatformSpecific with ZIOCompanionVersionSpecific
   )(implicit trace: Trace): ZIO[R, E, A] =
     Stateful(trace, onState)
 
+  private val MaxIterationsPerYield = 512
```

Comment:
So that the value is inlined

```suggestion
  private final val MaxIterationsPerYield = 512
```

Reply:
fixed